job-server/src/spark.jobserver/WebApi.scala

```diff
@@ -139,6 +188,21 @@ class WebApi(system: ActorSystem,
   }
 
   /**
+   * Routes for listing, deletion of and storing data files
+   *    GET /data                     - lists all currently stored files
```

Comment:
Just trying to get clarity. Will this survive server restart? 

1. Start job server
2. POST /data/file1, POST /data/file2
3. GET /data -> Expect two files file1 and file2
4. Restart jobserver
5. GET /data -> Can I expect the two file names to be returned?